### PR TITLE
makes a mini sentry riot turret

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
this code is the least modular thing I've ever seen what the heck
makes a riot turret with 50 rounds that stuns things and knocks them back, shoots semiautomatically.

## Why It's Good For The Game
adds a chep turret option that's more interesting

## Changelog
:cl:
add: Added new riot turret for marines, it stuns things but does low damage
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -69. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -981,6 +981,15 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	penetration = 10
 	sundering = 2
 
+/datum/ammo/bullet/turret/riot
+	name = "riot dart"
+	damage = 10
+	penetration = 100
+	sundering = 1
```
Comment (Hughgent) [inline code/modules/projectiles/ammo_datums.dm:988]: Make sure this does `STAMINA` damage

File: code/game/objects/machinery/sentries.dm [hunk 2]
```diff
@@ -1325,6 +1325,106 @@
 	new /obj/item/tool/screwdriver(src) //screw the gun onto the post.
 	new /obj/item/ammo_magazine/minisentry(src)
 
+
+
+// riot sentry
+
+/obj/machinery/marine_turret/mini/riot
+	name = "\improper UA-590 Riot Defense Sentry"
```
Comment (XSlayer300) [inline code/game/objects/machinery/sentries.dm:1333]:
```suggestion
	name = "\improper UA-590 riot defense sentry"
```

File: code/game/objects/machinery/sentries.dm [hunk 3]
```diff
@@ -1325,6 +1325,106 @@
...
+	max_burst = 1
+	rounds = 50
+	rounds_max = 50
+	knockdown_threshold = 100 //lighter, not as well secured.
+	work_time = 10 //significantly faster than the big sentry
+	ammo = /datum/ammo/bullet/turret/riot
+	magazine_type = /obj/item/ammo_magazine/minisentry/riot
+
+/obj/item/marine_turret/mini/riot
+	name = "\improper UA-590 Riot Defense Sentry (Folded)"
```
Comment (XSlayer300) [inline code/game/objects/machinery/sentries.dm:1353]:
```suggestion
	name = "\improper UA-590 riot defense sentry (folded)"
```

**Pull request closed without merging**